### PR TITLE
kubeadm: add instructions about rebalancing CoreDNS Pods after joining more nodes

### PR DESCRIPTION
**Pull request by SataQiu**
Fixes: kubernetes/kubeadm#2615

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -383,6 +383,12 @@ Run 'kubectl get nodes' on control-plane to see this machine join.
 A few seconds later, you should notice this node in the output from `kubectl get
 nodes` when run on the control-plane node.
 
+{{< note >}}
+As the cluster nodes are usually initialized sequentially, the CoreDNS Pods are likely to 
+run on the first control-plane node entirely. If you install a cluster with more than one node,
```
Comment (tengqm) [inline content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:388]: “entirely” in this context reads strange.

Comment (SataQiu) [reply content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md]: Thanks @tengqm 
Updated!

**Review by mehabhalodiya — COMMENTED** [review]
LGTM from SIG Docs!
/lgtm

File: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 2]
```diff
@@ -383,6 +383,12 @@ Run 'kubectl get nodes' on control-plane to see this machine join.
 A few seconds later, you should notice this node in the output from `kubectl get
 nodes` when run on the control-plane node.
 
+{{< note >}}
+As the cluster nodes are usually initialized sequentially, the CoreDNS Pods are likely to 
+all run on the first control-plane node. If you install a cluster with more than one node, 
+we recommend that you rebalance the CoreDNS Pods with `kubectl -n kube-system rollout restart deployment coredns`.
```
Comment (sftim) [inline content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:389]: nit ([avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we), and timing for when to do this):
```suggestion
rebalance the CoreDNS Pods with `kubectl -n kube-system rollout restart deployment coredns`
after you first add nodes that are not part of the control plane.
```

How about that?

Comment (SataQiu) [reply content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md]: Thanks @sftim 
Updated.

**Review by sftim — COMMENTED** [review]
/approve

**Comment by sftim** [comment]
I'll LGTM this as well. It does indeed look good to me.
/lgtm

**Pull request merged**